test/hotspot/jtreg/vmTestbase/nsk/jdi/ClassPrepareEvent/referenceType/refType001.java

```diff
@@ -257,17 +270,12 @@ public void run() {
                 testFailed = true;
             }
 
-            // notify EventHandler that all threads started
-            threadsStarted = true;
-
             // wait for all expected events received or timeout exceeds
             try {
-                  eventHandler.join(eventTimeout);
-                  if (eventHandler.isAlive()) {
-                      log.complain("FAILURE 20: Timeout for waiting event was exceeded");
-                      eventHandler.interrupt();
-                      testFailed = true;
-                  }
+                if (!eventsReceivedLatch.await(eventTimeout, TimeUnit.MILLISECONDS)) {
+                    log.complain("FAILURE 20: Timeout for waiting event was exceeded");
```

Comment:
Should it be "Timeout waiting for all events was exceeded" or maybe "Timeout while waiting for all events"? Same thing for thread001.java.

Reply:
Fixed.